src/main/java/hudson/plugins/svn_tag/SvnTagPlugin.java

```diff
@@ -170,6 +170,12 @@ public static boolean perform(AbstractBuild<?,?> abstractBuild,
                 logger.println(Messages.NoOldTag(evaledTagBaseURLStr));
             }
 
+            if(null != waitBeforeTagging && waitBeforeTagging > 0 )
+            {
+                logger.println(Messages.WaitBeforeTagging(waitBeforeTagging));
+                Thread.sleep(waitBeforeTagging*1000);
```

Comment:
Could you please clarify what is the bad practice here. So, I could be able fix it.